### PR TITLE
Release blog 2.10.4

### PR DESCRIPTION
**Pull request by liangyepianzhou**
[Google doc](https://docs.google.com/document/d/1RgmfoC6lbQjm7wrX_neKr3lw_-5ICHVW5UM1LKmy5Yk/edit#)
### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: blog/2023-04-19-Apache-Pulsar-2-10-4.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+In earlier versions, although users could create partitioned topics using the client when partitioned type auto-creation was enabled, there was no support for deleting these partitioned topics.
+
+#### Resolution
+This PR adds support for deleting partitioned topics using the keyword `-partition-`, making it easier for users to manage their partitioned topics.
+
+# What’s Next?
+
+If you are interested in learning more about Pulsar 2.10.4, you can [download](https://pulsar.apache.org/download/) and try it out now!
+
+For more information about the Apache Pulsar project and current progress, visit the Pulsar website and follow the project.
```
Comment (Anonymitaet) [inline blog/2023-04-19-Apache-Pulsar-2-10-4.md:59]:
```suggestion
For more information about the Apache Pulsar project and current progress, visit
the [Pulsar website](https://pulsar.apache.org), follow the project on Twitter
[@apache_pulsar](https://twitter.com/apache_pulsar), and join [Pulsar Slack](https://apache-pulsar.slack.com/)!
```

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution! Have you previewed your changes and ensured everything goes as expected? 

If not, please [preview your changes locally](https://pulsar.apache.org/contribute/document-preview/#preview-changes) and attach the screenshots to this PR. In this way, you can get your PR merged more quickly. Thanks!

**Comment by liangyepianzhou** [comment]
image

image

**Review by tisonkun — APPROVED** [review]
Thank you!

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**